### ISSUE TITLE
Color Board Touch Bar Widget Bug

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#1205]
Claudio Giovenzana [reports](https://www.facebook.com/groups/commandpost/permalink/2022342978021403/):

> Hello guys, last version of CP and FCPx installed. When using the Touch Bar Widget: Color Board (Grouped) as shown in the website (but configured for Final Cut Pro "Bar 2" in the preferences) i slide over the buttons with my fingers to adjust the parameters but the values (of color/saturation/exposure) flickers with some strange numbers and go back to zero... Any advices? Thank you

**Comment by latenitefilms** [comment]
I can confirm that the Color Board Touch Bar widget is no longer working - I'm ASSUMING that's because of the changes made to the FCPX Color Inspector API. I've had a quick play, and can't work out what's going wrong.

```lua
> _fcp:colorBoard():color():master():show():percent()
nil

> _fcp:colorBoard():saturation():master():show():percent()
nil

> _fcp:colorBoard():exposure():master():show():percent()
nil
```

**Comment by latenitefilms** [comment]
Fixed in #1156
